.github/workflows/testBuild.yml

```diff
@@ -17,19 +17,32 @@ jobs:
   validateActor:
     runs-on: ubuntu-latest
     outputs:
-      IS_TEAM_MEMBER: ${{ fromJSON(steps.isUserDeployer.outputs.isTeamMember) }}
+      READY_TO_BUILD: ${{steps.readyToBuild.outputs.READY_TO_BUILD}}
     steps:
-      - id: isUserDeployer
+      - id: isUserTeamMember
         uses: tspascoal/get-user-teams-membership@baf2e6adf4c3b897bd65a7e3184305c165aec872
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           username: ${{ github.actor }}
-          team: mobile-deployers
+          team: expensify-expensify
+      - name: Remove label if it was added by an unauthorized user
+        if: ${{ !fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && github.event.label.name == 'Ready To Build' }}
+        uses: actions-ecosystem/action-remove-labels@v1
+        with:
+          labels: 'Ready To Build'
+      - name: Throw exception if label was added by an unauthorized user
+        if: ${{ !fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && github.event.label.name == 'Ready To Build' }}
+        run: |
+          echo "The 'Ready To Build' label was added by an unauthorized user"
+          exit 1
+      - id: readyToBuild
+        name: Set READY_TO_BUILD flag
+        run: echo "READY_TO_BUILD=${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) || contains(github.event.pull_request.labels.*.name, 'Ready To Build') }}" >> "$GITHUB_OUTPUT"
 
   getBranchRef:
     runs-on: ubuntu-latest
     needs: validateActor
-    if: ${{ fromJSON(needs.validateActor.outputs.IS_TEAM_MEMBER) }}
+    if: ${{ needs.validateActor.outputs.READY_TO_BUILD == 'true' }}
```

Comment:
Out of curiosity, why not use `fromJSON` here?

Reply:
Good call @francoisl, using `fromJSON` is our preferred style